node-problem-detector.yaml

```diff
@@ -26,6 +26,11 @@ spec:
         - name: log
           mountPath: /log
           readOnly: true
+        # Make sure node problem detector is in the same timezone
+        # with the host.
+        - name: localtime
+          mountPath: /etc/localtime
+          readOnly: true
```

Comment:
FYI yesterday, Dan Walsh posted this on Atomic devel list (Atomic is basically Fedora):
https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2016-December/msg00014.html

In short, if the `/etc/localtime` files is are symbolic link, then you end up with the localtime in the container not updated.

However, I don't think you can do something to avoid this issue.